modules/getting-started-web-console-deploying-python-app.adoc

```diff
@@ -1,48 +1,66 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-deploying-python-app_{context}"]
-= Deploying a Python application
+= Deploying the back-end application
 
-The following procedure deploys a back-end service for the `parksmap` application. The Python application performs 2D geo-spatial queries against a MongoDB database to locate and return map coordinates of all national parks in the world.
-
-The deployed back-end service that is `nationalparks`.
+The following procedure deploys `nationalparks`, which is the back-end component for the `national-parks-app` application. The Python application performs 2D geo-spatial queries against a MongoDB database to locate and return map coordinates of all national parks in the world.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
-* You have a deployed image.
+* You have deployed the `parksmap` front-end application.
 
 .Procedure
 
-. From the *+Add* view in the *Developer* perspective, click *Import from Git* to open a dialog.
-. Enter the following URL in the Git Repo URL field:
-[x-]`https://github.com/openshift-roadshow/nationalparks-py.git`
+. From the quick create (+) dropdown in the upper right corner, click *Import from Git*.
+. In the *Git Repo URL* field, enter [x-]`https://github.com/openshift-roadshow/nationalparks-py.git`.
 +
-A builder image is automatically detected.
+A builder image is automatically detected, but the import strategy might default to Dockerfile instead of Python.
 +
-[NOTE]
-====
-If the detected builder image is Dockerfile, select *Edit Import Strategy*. Select *Builder Image* and then click *Python*.
-====
+// TODO: If it never detects it properly, update these steps to just always go through them (Remove the "If...")
```

Comment:
@AlexonOliveiraRH When I tested it, it didn't detect Builder Image / Python, it detected Dockerfile. When you test, can you see if yours also detects Dockerfile?

If that's what it's always going to detect, and they are always going to need to change it, I'll probably update this text some (line 20 and the "If..." on line 24).

Reply:
> @AlexonOliveiraRH When I tested it, it didn't detect Builder Image / Python, it detected Dockerfile. When you test, can you see if yours also detects Dockerfile?
> 
> If that's what it's always going to detect, and they are always going to need to change it, I'll probably update this text some (line 20 and the "If..." on line 24).

@bergerhoffer yep, mine detected `Dockerfile` as well. We need to change it everytime. Go ahead with the suggested edit, please.